validator/src/journal/chain.rs

```diff
@@ -528,8 +528,8 @@ impl<BC: BlockCache + 'static, BV: BlockValidator + 'static> ChainController<BC,
         info!("Ignoring block {}", block)
     }
 
-    // Returns all blocks in forks not on the chain with the given head. If head is None, uses the
-    // current chain head. If head is not found, returns None.
+    // Returns all valid blocks in forks not on the chain with the given head. If head is None,
+    // uses the current chain head. If head is not found, returns None.
```

Comment:
This comment is incorrect, but should be fixed in the associate PR against master.